frontend/src/views/web/landing.html

```diff
@@ -134,19 +134,20 @@ <h4><div>Who uses EvalAI?</div></h4>
                                </div>
```

Comment:
@anuj-rathore Why are you removing the `id` and why fb logo file is added again?